### PR TITLE
Reduxed SampleView with new GUI

### PR DESCRIPTION
**Pull request by bolmsten**
This PR contains the Reduxed SampleView with a new GUI. I have also made small changes to the rest of the CSS to correspond to the new GUI changes. 

This is a first implementation of the new SampleView and it uses the Fabric JS library to draw on the Canvas. This seems to be the best library for this use at the moment but it does not build correctly in npm so it was added manually. It seems that they are working on fixing this issue so it will be included into the npm build later.

At the moment you can right click on points and see what actions are available. You can also select two points and draw a line. All this corresponds to the discussed functionality in Barcelona.

### CODE REVIEW

File: mxcube3/ui/components/Login/Login.jsx [hunk 1]
```diff
@@ -28,15 +24,17 @@ class Login extends React.Component {
   signIn(){
       let fields = this.props.fields;
       this.props.signIn(fields.username.value, fields.password.value);
-      this.setState({ show_loader: true });
+      $("#ajax_loader").toggleClass('hidden',false);
```
Comment (mguijarr) [inline mxcube3/ui/components/Login/Login.jsx]: I guess there is a regression here... jQuery is back.
A merge failed maybe ?

**Comment by mguijarr** [comment]
Well done !

I will merge it as soon as the jQuery problem is fixed (see my comment in code)

**Comment by bolmsten** [comment]
It should be reverted now

**Comment by bolmsten** [comment]
Good catch

File: mxcube3/ui/reducers/index.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import login from './login'
 import queue from './queue'
 import samples_grid from './samples_grid'
 import methodForm from './methodForm'
+import sampleView from './sampleView'
```
Comment (JieNanMAXIV) [inline mxcube3/ui/reducers/index.js]: should be sampleview, as the file sampleView.js does not exist.

Comment (mguijarr) [inline mxcube3/ui/reducers/index.js]: Yes! You're completely right

**Pull request merged**